File: app/src/main/java/huitca1212/cuantotemide/OptionsActivity.java
package huitca1212.cuantotemide;

import android.support.v7.app.AppCompatActivity;
import android.view.Menu;
import android.view.MenuItem;

public class OptionsActivity extends AppCompatActivity {

	@Override
	public boolean onCreateOptionsMenu(Menu menu) {
		getMenuInflater().inflate(R.menu.main, menu);
		return true;
	}

	@Override
	public boolean onOptionsItemSelected(MenuItem item) {
		switch (item.getItemId()) {
			case R.id.menu_share:
				Utils.share(this);
				return true;
			case R.id.menu_info:
				Utils.showInfoAppDialog(this);
				return true;
		}
		return true;
	}
}

File: app/src/main/java/huitca1212/cuantotemide/OptionsActivity.kt
package huitca1212.cuantotemide

import android.view.Menu
import android.view.MenuItem
import androidx.appcompat.app.AppCompatActivity

open class OptionsActivity : AppCompatActivity() {

    override fun onCreateOptionsMenu(menu: Menu): Boolean {
        menuInflater.inflate(R.menu.main, menu)
        return true
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        when (item.itemId) {
            R.id.menu_share -> Utils.share(this)
            R.id.menu_info -> Utils.showInfoAppDialog(this)
        }
        return true
    }
}
